src/Marten/Events/Projections/ProjectionOptions.cs

```diff
@@ -161,7 +161,6 @@ public MartenRegistry.DocumentMappingExpression<T> SelfAggregate<T>(ProjectionLi
         /// </summary>
```

Comment:
We put the `null` value here instead of a single one because different projections have different defaults. For instance, single stream projections are by default inline, while multistream is async (as there's more processing there). Having `null` here means: _Use the default from the projection type_. It's a good recommendation, but it'd require more changes; maybe we should add more explicit registration. @jeremydmiller FYI.

Reply:
Thanks for clarification!

As consumer of Marten I don't think more explicit registrations are needed. Having "Use the default from the projection type" in the doc-comment would be sufficient. Maybe in addition with "single stream projections are by default inline, while multistream is async" being in the docs.
I assumed that it is this way (at least I would have implemented it like this 😉), but couldn't find info for this.

Reply:
Yeah, that's fair; thanks for taking care of that; that's much appreciated!